AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -366,7 +366,7 @@
 
 
 /**
- * Called just before the view is added to a superview.
+ * Called just before the view is added to a window.
```

Comment:
Sadly I think the behavior of this method is subtle.  It actually varies a bit for layer-backed nodes, which rely on kCAOrderIn (CAAction) rather than the UIView-unique hierarchy notifications.  This needs better documentation, and in fact, I'm pretty sure a Github issue is filed for this purpose.